### PR TITLE
Strings

### PR DESCRIPTION
**Pull request by ferueda**
String article and tests (tasks and solutions) translated to spanish.

### CODE REVIEW

File: 1-js/05-data-types/03-string/2-check-spam/solution.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ function checkSpam(str) {
   return lowerStr.includes('viagra') || lowerStr.includes('xxx');
 }
 
-alert( checkSpam('buy ViAgRA now') );
-alert( checkSpam('free xxxxx') );
-alert( checkSpam("innocent rabbit") );
+alert( checkSpam('compra ViAgRA ahora') );
+alert( checkSpam('xxxxx gratis') );
+alert( checkSpam("coneja inocente") );
```
Comment (spideep) [inline 1-js/05-data-types/03-string/2-check-spam/solution.md]: 🐰

File: 1-js/05-data-types/03-string/article.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
 # Strings
 
-In JavaScript, the textual data is stored as strings. There is no separate type for a single character.
+En JavaScript, los datos textuales son almacenados como strings (cadena de caracteres). No hay un tipo de datos separado para caracteres únicos.
```
Comment (spideep) [inline 1-js/05-data-types/03-string/article.md]: 'unitarios' en lugar de 'únicos'. Puesto que se refiere a la cantidad de caracteres no a la exclusividad.

File: 1-js/05-data-types/03-string/2-check-spam/solution.md (hunk 1 above)
Comment (ferueda) [reply 1-js/05-data-types/03-string/2-check-spam/solution.md]: No entiendo el cambio solicitado aquí.

**Comment by ferueda** [comment]
It looks like no one is reviewing these pull requests from the community. If you need help revewing them I can help as a reviewer.

**Pull request merged**